src/main/java/seedu/address/logic/parser/ListCommandParser.java

```diff
@@ -42,20 +41,15 @@ public ListCommand parse(String args) throws ParseException {
 
         Set<Tag> tagList = isTagFlagPresent ? ParserUtil.parseTags(argMultimap.getAllValues(PREFIX_TAG)) : null;
         Set<Amenity> amenityList = isAmenityFlagPresent
-                ? ParserUtil.parseAmenities(argMultimap.getAllValues(PREFIX_AMENITY))
-                : null;
-
-        Rating rating = isRatingFlagPresent
-                ? ParserUtil.parseRating(argMultimap.getValue(PREFIX_RATING).get()) : null;
+                ? ParserUtil.parseAmenities(argMultimap.getAllValues(PREFIX_AMENITY)) : null;
+        Rating rating = isRatingFlagPresent ? ParserUtil.parseRating(argMultimap.getValue(PREFIX_RATING).get()) : null;
 
```

Comment:
In my opinion the previous ternary is clearer to read (because of linebreaks), and I feel that length of method here can be sacrificed in place of clarity...?